daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -94,7 +94,7 @@ object Daffodil {
    * @return new object to compile DFDL schemas
    */
   def compiler(): Compiler = {
-    new Daffodil // silence warning about unused private constructor
+    val _ = new Daffodil // silence warning about unused private constructor
```

Comment:
If we remove this does it still do the warning about unused private constructor? Maybe this isn't needed anymore?

Reply:
No,, it doesn't complain anymore. I'll remove the lines